wsinfer/cli/convert_csv_to_geojson.py

```diff
@@ -30,7 +30,7 @@ def _row_to_geojson(row: pd.Series, prob_cols: list[str]) -> dict:
     measurements = [{"name": k, "value": v} for k, v in prob_dict.items()]
     return {
         "type": "Feature",
-        "id": "PathTileObject",
+        "id": "49069bbb-a0ac-4343-96fe-15af45c30634",
```

Comment:
instead of setting a single id, let's generate a new one for each instance. these ids are [UUIDs](https://en.wikipedia.org/wiki/Universally_unique_identifier) 

you will also have to add `import uuid` (it's part of the standard library)

```suggestion
        "id": str(uuid.uuid4()),
```